platform/ios/PolyPodApp/PodApi/PolyOut/FS/PolyOut+FS.swift

```diff
@@ -234,12 +234,13 @@ extension PolyOut {
     
     func removeArchive(fileId: String, completionHandler: (Error?) -> Void) {
         do {
-            let path = fsUriFromId(fileId).path
+            let path = fsUriFromId(idFromPodUrl(fileId)!).path
```

Comment:
Yeah, let's not force unwrap here. It can lead to crashes.

Reply:
Which are caught and property reported? What's wrong about that?

Reply:
It won't be caught. It will crash the app, even if you wrap it in a do catch block.